### ISSUE TITLE
Alt text cut off vertically when there is not enough space.

### ISSUE THREAD

**Issue by babeuh** [mastodon/mastodon-android#595]
This seems to happen with small images like on this tweet: https://floss.social/@moshidon/110515421002038182

**Comment by grishka** [comment]
It is scrollable though

**Comment by babeuh** [comment]
Yes, it is, I didn't notice that. But I still think it's not great, the close button is cut off, the scrollbar is difficult to see and you have to put your finger at the perfect place to scroll.

If you don't think this is very necessary to fix (improve), feel free to close the issue.

**Comment by grishka** [comment]
Maybe it would benefit from having the "fading edge" thing enabled, as an affordance to show that it's scrollable 🤔

I'll bring this up with our designer.
